00DIST

```diff
@@ -5295,11 +5295,14 @@ July 14, 2018
 		found
 		submitted by @emaste
 
-               Fix broken LSOF_CFLAGS_OVERRIDE.
-               Provided by Fabrice Fontaine in #172.
+		Fix broken LSOF_CFLAGS_OVERRIDE.
+		Provided by Fabrice Fontaine in #172.
 
-               Fix broken AR and RANLIB when cross-compiling.
-               Provided by Fabrice Fontaine in #194.
+		Fix broken AR and RANLIB when cross-compiling.
+		Provided by Fabrice Fontaine in #194.
```

Comment:
Could you make a separate commit for this "docs: convert spaces to tabs" change?
This change is nothing to do with "dialects/linux/machine: remove sysvlegacy function ".